lib/bibdata_rs/src/ephemera/ephemera_folder.rs

```diff
@@ -502,4 +572,44 @@ mod tests {
 
         assert_eq!(result.thumbnail_url, "https://iiif-cloud.princeton.edu/iiif/2/c9%2Fa6%2F2b%2Fc9a62b81f8014b13933f4cf462c092dc%2Fintermediate_file/full/!200,150/0/default.jpg".to_string());
     }
+
+    #[test]
+    fn it_does_not_error_on_invalid_origin_place_type() {
+        use serde_json::json;
+
+        let invalid_json = json!({
+            "@id": "test-id",
+            "title": ["Test Title"],
+            "origin_place": [""]
+        });
+
+        let result: Result<EphemeraFolder, _> = serde_json::from_value(invalid_json);
+
+        assert!(
+            result.is_ok(),
+            "Deserialization should not error on invalid origin_place type"
+        );
+        let folder = result.unwrap();
+        assert!(folder.origin.is_none() || folder.origin.as_ref().unwrap().0.is_empty());
```

Comment:
This always is `None` for me -- would it be okay to simplify the assertion (and the one in the next test as well)?
```suggestion
        assert!(folder.origin.is_none());
```